### DISCUSSION TITLE
installation-guide/dev-env/nvm-nodejs

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#2] [Announcements]
# installation-guide/dev-env/nvm-nodejs

Node.js开发环境配置 记得DD第一次接触Node.js还是0.10.x版本，在2016年的时候，还写过一篇关于Node.js开发环境的搭建，当时版本是5.3.x。而到写这篇的时候，都已经20.8.0了。 现在关于Node.js的框架和工具也已经越来越全了，要搭建一个简单的Node.js开发环境也是轻而易举。直接通过官网（ https://nodejs.org/）下载对应平台的安装包，一直下一步就能完成。

https://www.didispace.com/installation-guide/dev-env/nvm-nodejs.html

**Comment by dyc87112** [comment]
1
